packages/avatars/src/styled/StyledStandaloneStatusCaption.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme, css } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'avatars.status-indicator.caption';
+
+function sizeStyles(props: ThemeProps<DefaultTheme>) {
+  return css`
+    padding: 0 ${props.theme.space.base}px;
```

Comment:
Following the previous suggestion, the caption should one-way margin (not padding) the entire 8px depending on RTL. Please see the RFC codesandbox for basic layout details.

Reply:
I added the RTL portion to a TODO that I will look at. The disadvantage of this approach is that flex containers will flow correctly with direction without the need to direct the margin based on RTL. Even spacing allows elements to flow naturally IMO.